src/ISA/ISA/ARCTypes/ARCTable.fs

```diff
@@ -1,12 +1,69 @@
 ﻿namespace ISA
 
+
 type ArcTable = 
     {
         Name : string
         ValueHeaders : CompositeHeader []
+        /// Key: Column * Row
         Values : System.Collections.Generic.Dictionary<int*int,CompositeCell>  
     }
 
+    /// Create ArcTable with empty 'ValueHeader' and 'Values' 
+    static member init(name: string) = {
+        Name = name
+        ValueHeaders = [||]
+        Values = System.Collections.Generic.Dictionary<int*int,CompositeCell>()
+    }
+
+    member this.ColumnCount = this.ValueHeaders.Length
+    member this.RowCount = this.Values.Keys |> Seq.maxBy snd |> snd
+
+    member this.addColumn (header:CompositeHeader, ?cells: CompositeCell [], ?index: int) = 
+        let cells = Option.defaultValue [||] cells
+        let column = Array.singleton <| CompositeColumn.create(header, cells)
+        this.addColumns (column, ?index = index)
+
+    member this.addColumns ( columns: CompositeColumn [], ?index: int) = 
+        let index = Option.defaultValue this.ColumnCount index
+        columns |> Array.iter (fun x -> x.validate(true) |> ignore)
+        let nNewColumn = columns.Length
+        let columnHeaders, columnRows = columns |> Array.map (fun x -> x.Header, x.Cells) |> Array.unzip
+        let nextHeaders = 
+            let ra = ResizeArray(this.ValueHeaders)
+            ra.InsertRange(index, columnHeaders)
+            ra.ToArray()
+        let maxRows = 
+            let maxRows = columnRows |> Array.map (fun x -> x.Length) |> Array.max
+            System.Math.Max(this.RowCount, maxRows)
+        let nextBody =
```

Comment:
Should we make the members mutable instead? This could boost performance and would probably follow expectations (See behaviour of Deedle, where member do mutable operations on the frame. Static members work immutable there.).

@muehlhaus